anchore_engine/analyzers/manager.py

```diff
@@ -29,8 +29,8 @@ def apply_hints(analyzer_report, unpackdir):
     # the config does not explicitly enable hints processing, effectively disabling
     # hints processing.
     for engine_entry in utils.content_hints(unpackdir=unpackdir):
-        pkg_type = engine_entry.get("type")
-        if pkg_type:
+        pkg_type = engine_entry.get("type", "")
```

Comment:
dictionary .get() returnes `None` if the key doesn't exist. Is there any particular reason why the default value is an empty string?

Reply:
fixed